version.sbt

```diff
@@ -1 +1 @@
-ThisBuild / version := "2.1.2-SNAPSHOT"
```

Comment:
You don't usually change this one manually (you don't have to at least). Instead, after merging to `master`, you can invoke `sbt release` from `master` branch (possibly by adding `skip-tests` if you're on Apple Silicon and tests are using Cassandra from `cassandra-launcher` which is not supported yet). It will then ask you to enter the version being released while trying to deduce and suggest it (it's going to suggest `2.1.2` in our case, but you're going to enter `3.0.0`) and ask to enter the next snapshot version (it's going to suggest `3.0.1-SNAPSHOT` here which is fine with us). Then it will automatically create new commits with `version.sbt` updates and ask you if it can push them to the repository along with the tag. I recommend agreeing to push commits and tags - less chance of a mistake. Tags mark releases and then you can auto-generate release notes on the `Releases` page from them.